src/BackgroundTask/MonitorCssTransientCaching.php

```diff
@@ -145,6 +155,18 @@ public static function query_css_transient_count() {
 		);
 	}
 
+	/**
+	 * Handle update to plugin.
+	 *
+	 * @param string $old_version Old version.
+	 */
+	public function handle_plugin_update( $old_version ) {
+		// Reset the disabling of the CSS caching subsystem when updating from versions 1.5.0 or 1.5.1.
+		if ( version_compare( $old_version, '1.5.0', '>=' ) || version_compare( $old_version, '1.5.2', '<' ) ) {
```

Comment:
I think `&&` is needed here, no?

```suggestion
		if ( version_compare( $old_version, '1.5.0', '>=' ) && version_compare( $old_version, '1.5.2', '<' ) ) {
```